osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/pom.xml

```diff
@@ -267,6 +265,12 @@
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-engine</artifactId>
     </dependency>
+      <!-- Cucumber tests depend on JUnit 4, add the vintage engine -->
+      <dependency>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=xml%3AIndentCheck)